crowdfunding/users/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns =[
+    path('', views.CustomUserList.as_view(), name='customuser-list'),
+    path('<int:pk>', views.CustomUserDetail.as_view(), name='customuser-detail'),
+    path('<int:pk>/change-password/', views.ChangePasswordView.as_view(), name='change-password'),
```

Comment:
nice url

Reply:
you probably don't need the `<int:pk>` part of this as you aren't looking up that user id in your view